serializers/src/stl/stlSerializer.ts

```diff
@@ -37,21 +45,85 @@ export class STLExport {
                 data += '\t\tvertex ' + v[2].x + ' ' + v[2].y + ' ' + v[2].z + '\r\n';
                 data += '\tendloop\r\n';
                 data += 'endfacet\r\n';
-            }
-            data += 'endsolid exportedMesh';
-
-            if (download) {
-                let a = document.createElement('a');
-                let blob = new Blob([data], {'type': 'application/octet-stream'});
-                a.href = window.URL.createObjectURL(blob);
-
-                if (!fileName) {
-                    fileName = "STL_Mesh";
-                }
-                a.download = fileName + ".stl";
-                a.click();
-            }
-
-            return data;
+            }			
+		}	
+		
+		data += 'endsolid exportedMesh';
+		
+		}else{	
+		
+			//Adapted from https://gist.github.com/paulkaplan/6d5f0ab2c7e8fdc68a61			
+			let faceCount = 0;
```

Comment:
identation seems off all over the file?

Reply:
It does look odd on here which is crazy because on my local version it looks fine.
![image](https://user-images.githubusercontent.com/1538857/53427464-d3593b80-399d-11e9-8962-e600cd57df1a.png)

Reply:
I'm not sure how else to indent it?

Also do not merge it yet! I just found a error.

Reply:
Please use 4 spaces (does not seem the case here)

Reply:
Ahh, ok yeah I was using tabs. Ill fix asap.

Reply:
:D